res/openvpn-gui-res-it.rc

```diff
@@ -332,127 +347,126 @@ Completa la richiesta di autorizzazione precedente."
+    status cnn           \t\t: visualizza lo stato della configurazione ""cnn"" se connessa\n\
     silent_connection [0|1]\t: imposta la flag silent_connection on (1) oppure off (0)\n\
-    import path          \t\t: Import the config file pointed to by path\n\
+    import path          \t\t: Importa il file di configurazione puntato dal percorso\n\
 \t\t\tEsempio: openvpn-gui.exe --command disconnect myconfig\n\
 \n\
-Opzioni per ignorare il registro di sistema:\n\
---exe_path\t\t: Percorso di openvpn.exe.\n\
---config_dir\t\t: Percorso della cartella dei file di configurazione.\n\
---ext_string\t\t: Estensione dei file di configurazione.\n\
---log_dir\t\t\t: Percorso della cartella dei file di log.\n\
---priority_string\t\t: Stringa di priorità (Vedi install.txt per maggiori informazioni).\n\
---append_string\t\t: 1=Aggiungi in coda al log. 0=Tronca log alla connessione.\n\
---log_viewer\t\t: Percorso del visualizzatore log.\n\
---editor\t\t\t: Percorso dell'editor di configurazione.\n\
---allow_edit\t\t: 1=Mostra opzione modifica configurazione.\n\
---allow_service\t\t: 1=Mostra menu controllo servizio.\n\
---allow_password\t\t: 1=Mostra menu modifica password.\n\
---allow_proxy\t\t: 1=Mostra menu impostazioni proxy.\n\
---show_balloon\t\t: 0=Mai, 1=Alla connessione iniziale, 2=A ogni riconnessione.\n\
+Opzioni per sovrascrivere le impostazioni:\n\
+--exe_path\t\t: Percorso openvpn.exe.\n\
+--config_dir\t\t: Percorso cartella file configurazione.\n\
+--ext_string\t\t: Estensione file configurazione.\n\
+--log_dir\t\t\t: Percorso cartella file registro eventi.\n\
+--priority_string\t\t: Stringa priorità (per maggiori informazioni vedi 'install.txt').\n\
+--append_string\t\t: 1=Aggiungi in coda al registro eventi. 0=Tronca registro alla connessione.\n\
+--log_viewer\t\t: Percorso visualizzatore registro eventi.\n\
+--editor\t\t\t: Percorso editor di configurazione.\n\
+--allow_edit\t\t: 1=Visualizza opzione modifica configurazione.\n\
+--allow_service\t\t: 1=Visualizza menu controllo servizio.\n\
+--allow_password\t\t: 1=Visualizza menu modifica password.\n\
+--allow_proxy\t\t: 1=Visualizza menu impostazioni proxy.\n\
+--show_balloon\t\t: 0=Mai, 1=Alla connessione iniziale, 2=Ad ogni riconnessione.\n\
 --service_only\t\t: 1=Abilita la modalità solo servizio.\n\
---silent_connection\t\t: 1=Non mostrare la finestra di stato durante la connessione e avvertimenti non urgenti all'avvio.\n\
---show_script_window\t: 0=Nascondi la finestra di esecuzione dello script, 1=Mostra.\n\
---passphrase_attempts\t: Numero di tentativi permessi per la frase di sicurezza.\n\
+--silent_connection\t\t: 1=Non visualizzare la finestra di stato durante la connessione e avvertimenti non urgenti all'avvio.\n\
+--show_script_window\t: 0=Nascondi la finestra di esecuzione dello script, 1=Visualizza.\n\
+--passphrase_attempts\t: Numero di tentativi permessi per la password.\n\
```

Comment:
is this specific to the private key passphrase? if so, it could be specified

Reply:
ok if if passphrase is not password.